package.json

```diff
@@ -89,6 +89,7 @@
     "chart.js": "~2.9.4",
     "core-js": "~3.6.4",
     "cron-parser": "~4.2.1",
+    "cron-validator": "~1.3.1",
```

Comment:
We can keep it, but it was possible to use `cron-parser` to check if cron is valid. See `CrontabPartValidatorService`.

Reply:
I'd like to keep cron-validator, since all it is only responsible for is - cron validation, returning `:boolean`, which I used in other places as well.

Reply:
@undsoft - I removed `cron-validator`